File: csv_tdms_read/csv/TempPlot_csvread.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Sep 18 16:09:38 2017

@author: Sujay
"""
import numpy as np
import matplotlib
import matplotlib.pyplot as plt

data = np.genfromtxt('b141511.csv', delimiter=',')
data = np.delete(data, (0), axis=0)

TMP = data[:, 0]
LED = data[:, 1]
PD = data[:, 2]
time = np.linspace(0, 162, num=163)

plt.subplot(3, 1, 1)
plt.plot(time, TMP, 'r')
plt.title('TMP (deg C) vs Time (mins)')

plt.subplot(3, 1, 2)
plt.plot(time, LED, 'b')
plt.title('LED voltage (V) vs Time (mins)')

plt.subplot(3, 1, 3)
plt.plot(time, PD, 'b')
plt.title('PDout (V) vs Time (mins)')

File: csv_tdms_read/csv/TempPlot_csvread.m
clear all; close all;
plotfigs = 1;

X = csvread('b141511.csv',1,0); 
TMP=X(:,1);
LED=X(:,2);
PD=X(:,3);

% plot(TMP,'r');
% hold on;
% plot(LED,'b');
% hold on;
% plot(PD,'b');
% hold off; 
% title('TMP, LED, PD vs Time');
% xlabel('Time');
% ylabel('Volts or degC');
% legend('TMP','LED','PD');
figure();
subplot(3,1,1);
plot(TMP,'r');
title('TMP (degC) vs Time (mins)');
subplot(3,1,2);
plot(LED,'b');
title('LED voltage (V) vs Time (mins)');
subplot(3,1,3);
plot(PD,'b');
title('PDout (V) vs Time (mins)');